### DISCUSSION TITLE
Missing type DotNetCoreTestSettings from Cake.Common assembly

### DISCUSSION THREAD

**Discussion by sandeep-chinni** [cake-build/cake#4106] [Q&A] [answered]
I installed 3.0.0 version of Cake and when I run the command to execute my cake file, I get the below error. The first three are warnings and the last one is a error. I had to downgrade to 2.2.0 to make this error go away.

The assembly 'Cake.Coveralls, Version=0.10.1.0, Culture=neutral, PublicKeyToken=null' 
is referencing an older version of Cake.Core (0.33.0). 
For best compatibility it should target Cake.Core version 3.0.0.
The assembly 'Cake.Incubator, Version=5.1.0.0, Culture=neutral, PublicKeyToken=null' 
is referencing an older version of Cake.Core (0.33.0). 
For best compatibility it should target Cake.Core version 3.0.0.
Error: Could not load type 'Cake.Common.Tools.DotNetCore.Test.DotNetCoreTestSettings' from assembly 'Cake.Common, Version=3.0.0.0, Culture=neutral, PublicKeyToken=null'.

Has anyone faced this issue and resolved it?

**Comment by pascalberger** [comment, marked as answer]
`DotNetCore*` aliases have been removed in Cake 3.0.0 and replaced with `DotNet*` aliases. See [Upgrade instructions](https://cakebuild.net/docs/getting-started/upgrade#dotnet-aliases)

**Reply by sandeep-chinni** [reply]
@pascalberger I replaced all instances of DotNetCore* with DotNet* in my build.cake file, uninstalled 2.2.0 and installed 3.0.0 and I am still getting the same error.

**Reply by pascalberger** [reply]
It's hard to say exactly without further information. 

The warnings mention that you use addins which are built against a very old version of Cake. There's is no guarantee that they are working with this version of Cake and it can be that one of them makes use of no longer available APIs. You should update them to a compatible version and try again.

**Reply by sandeep-chinni** [reply]
@pascalberger Thanks for that. I had these references at the top of my build.cake (copy over from another file). I removed them and ran the build again and that was successful.

#tool "nuget:https://www.nuget.org/api/v2?package=coveralls.io&version=1.4.2"
#addin "nuget:https://www.nuget.org/api/v2?package=Cake.Coveralls&version=0.10.1"
#addin "nuget:https://www.nuget.org/api/v2?package=Cake.Incubator&version=5.1.0"

**Reply by pascalberger** [reply]
If you don't need these tools and addins you can remove them.

Otherwise it is probably the Cake.Incubator addin, which references `DotNetCoreTestSettings`. You have used version 5.1 which supports Cake 0.33. Latest version is 7.0 which supports Cake 2.x. You can track https://github.com/cake-contrib/Cake.Incubator/issues/216 for updating it to Cake 3.x if you need it.
